### ISSUE TITLE
systemctl example

### ISSUE THREAD

**Issue by ziggie216** [stevejenkins/pihole-cloudsync#36]
For the README.me, Automating with systemd, systemd Details, .service its show this line
ExecStart=/usr/local/bin/pihole-cloudsync --pull

shouldn't it be?
ExecStart=/usr/local/bin/pihole-cloudsync/pihole-cloudsync --pull

**Comment by mtijas** [comment]
Yes it indeed should be `ExecStart=/usr/local/bin/pihole-cloudsync/pihole-cloudsync --pull`. I just set up pihole-cloudsync (seems like a great project btw!) and didn't notice the "wrong" command at first (just copied the example from the README). After fixing the ExecStart command as you described above, systemd service started working.

Came here to report this issue and saw it's already reported, so just commenting/confirming this here.

**Comment by leshacat** [comment]
Yes it is, I found out before seeing this and made a PR https://github.com/stevejenkins/pihole-cloudsync/pull/38

You can also under **[service]** add:
```
User=root
Group=root
```
